packages/frontend/src/document/model_document_editor.css

```diff
@@ -8,15 +8,89 @@
     justify-content: space-between;
 }
 
-.model-theory select {
-    appearance: none;
+.selector {
+    text-transform: capitalize;
     border: none;
+    font-size: 17px;
+    padding: 16px;
     background-color: transparent;
-    color: var(--main-color);
-    font-size: var(--h3-font-size);
-    text-align-last: right;
+    font-family: var(--main-font);
+    cursor: pointer;
+    font-weight: 600;
 }
 
-.model-theory select:invalid {
-    color: darkgray;
+.selector:hover {
+    color: #53b6b2;
+}
+
+.description {
+    color: rgb(106, 106, 106);
+    font-size: 11px;
+    text-align: left;
+}
+
+#input-selections input[type="radio"] {
+    position: absolute;
+    opacity: 0;
+}
+
+#input-selections {
+    display: flex;
+    flex-direction: column;
+    margin: 5px;
+    background-color: transparent;
+    text-align: left;
+    border-top: transparent;
+    border-left: transparent;
+}
+
+#selection-items:hover {
+    color: #53b6b2;
+    cursor: pointer;
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+}
+
+ul li {
+    margin: 5px;
+    padding: 6px;
+}
+
+#division-category {
+    font-weight: 600;
+    padding-bottom: 1px;
+    color:  #53b6b2;
+}
+
+.overlay {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(0, 0, 0, 0.5);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    z-index: 1000;
+}
+
+.popup {
```

Comment:
A class called `popup` is already defined in `index.css`: it's confusing, and possibly even ill-defined, which one will take precedence.

1. We should have generic class `popup`, to be used throughout the app for consistent styling of popups. That should be defined in `index.css`
2. Any styling specific to this widget can be defined here, but if needed it should be qualified somehow (see above)